src/Server/TaskRequest.php

```diff
@@ -188,9 +192,9 @@ public function requireQueryParam(string $param): string
      * The exception short-circuits further execution but will be mapped to an HTTP response
      * that tells CloudTasks not to bother retrying.
      *
-     * @param string $param
+     * @param string $key
      *
-     * @return string
+     * @return string|array
```

Comment:
I guess now that we're parsing JSON, this can in fact be absolutely anything?

Reply:
Yeah right enough. Was thinking it's only one level deep but it could still have ints or floats etc in the top level. I'll change it to mixed